### ISSUE TITLE
work with neogit

### ISSUE THREAD

**Issue by zhengpd** [rhysd/committia.vim#59]
Hi, does anyone know how to set it up with neogit? Commiting from neogit does not trigger committia.

**Comment by cultab** [comment]
I'm currently looking into this as I want to do the same thing, seems that commitia's autocmd is not being run when openning neogit's `COMMIT_EDITMSG` buffer.

**Comment by rhysd** [comment]
I'm not user of neogit so I don't know how it works. committia.vim is triggered when `COMMIT_EDITMSG` or `MERGE_MSG` files are edited. To trigger manually, call `committia#open('git')`.

https://github.com/rhysd/committia.vim/blob/0b4df1a7f48ffbc23b009bd14d58ee1be541917c/plugin/committia.vim#L13

If neogit opens a commit buffer with other names, the following configuration may solve the issue:

```vim
autocmd BufReadPost file_name_opened_by_neogit_here call committia#open('git')
```

**Comment by cultab** [comment]
> committia.vim is triggered when `COMMIT_EDITMSG` or `MERGE_MSG` files are edited **on vim starting only**.
> 
shouldn't `vim.g.committia_open_only_vim_starting = 0` make committia open regardless of if vim is starting or not, did I missunderstand the following? https://github.com/rhysd/committia.vim/blob/0b4df1a7f48ffbc23b009bd14d58ee1be541917c/plugin/committia.vim#L8?

**Comment by rhysd** [comment]
@cultab You're correct. Doesn't it work on your machine?

**Comment by cultab** [comment]
@rhysd not exactly. Opening vim and then running `e COMMIT_EDITMSG` does not work. I don't even think the autocmd fires.
